config/initializers/devise.rb

```diff
@@ -260,7 +260,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :github, Figaro.env.gh_client_id, Figaro.env.gh_client_secret, scope: 'user:email'
+  config.omniauth :github, Figaro.env.gh_client_id, Figaro.env.gh_client_secret, display: 'popup', scope: 'user:email'
+  config.omniauth :stackexchange, client_id: Figaro.env.so_client_id, client_secret: Figaro.env.so_client_secret, public_key: Figaro.env.so_key, display: 'popup'
```

Comment:
This is, without a doubt, the weirdest line we wrote. But thankfully it works. I need to figure out what `display: 'popup'` is actually doing. I'm guessing not a thing.